babel/src/main/java/org/apache/calcite/sql/babel/SqlExecMacro.java

```diff
@@ -32,31 +33,63 @@
 import java.util.Objects;
 
 /**
- * Parse tree for {@code EXEC} statement.
+ * Parse tree for {@code SqlExecMacro} statement.
  */
 public class SqlExecMacro extends SqlCall implements SqlExecutableStatement {
   public static final SqlSpecialOperator OPERATOR =
       new SqlSpecialOperator("EXECUTE", SqlKind.OTHER);
```

Comment:
While we're at it, let's create a distinct SqlKind for EXECUTE.

Reply:
I have added EXECUTE to the set of DML operator list. Base on the name EXEC MACRO, I believe it would fall in to the DML category. https://github.com/googleinterns/calcite/pull/108/commits/52b2158d1128394efb3ef1a7f723e3f23fecc648